lib/interfaces/bdd.js

```diff
@@ -88,7 +95,18 @@ module.exports = function (suite) {
         fn = opts;
         opts = {};
       }
-
+      if (!afterAllHooksAreLoaded) {
+        afterAllHooks.forEach(function (hook) {
```

Comment:
I added comment in test.
FYI, we collect all hooks in right order in array and then push them to suite. We start collecting hooks from helpers and after from testfile. this cause hooks reordering 
(expected: `After` from test -> `_after` hooks from helpers -> `AfterSuite` from test -> `_afterSuite` hook from helpers.
actual:  `_after` hooks from helpers -> `After` from test -> `_afterSuite` hook from helpers. -> `AfterSuite` from test )

This change fix this chain